generators/chipyard/src/main/scala/TileOnlyHarnessBinders.scala

```diff
@@ -0,0 +1,171 @@
+package chipyard.harness
+
+import chisel3._
+
+import chipyard._
+import freechips.rocketchip.tilelink._
+import freechips.rocketchip.tile.NMI
+import org.chipsalliance.cde.config._
+import freechips.rocketchip.util.HeterogeneousBag
+
+import testchipip._
+
+
+
+class WithTileMasterHarness extends OverrideHarnessBinder({
+  (system: HasTileMasterPort, th: HasHarnessSignalReferences, ports: Seq[HeterogeneousBag[TLBundle]]) => {
```

Comment:
Similarly, for HarnessBinders, if you get rid of IOBinders you can flatten all of this into TileOnlyTestHarness.